src/Compilers/Core/Portable/Syntax/SyntaxNode.cs

```diff
@@ -348,6 +348,17 @@ public bool IsEquivalentTo(SyntaxNode other)
             return this.Green.IsEquivalentTo(other.Green);
         }
 
+        /// <summary>
+        /// Returns true if these two nodes are considered "incrementally identical".  An incrementally identical node
+        /// occurs when a <see cref="SyntaxTree"/> is incrementally parsed using <see cref="SyntaxTree.WithChangedText"/>
+        /// and the incremental parser is able to take the node from the original tree and use it in its entirety in the
+        /// new tree.  In this case, the <see cref="SyntaxNode.ToFullString()"/> of each node will be the same, though 
+        /// will have different parents, and may occur at different positions in the respective trees.  If two nodes are
```

Comment:
The way this comment is worded, it implies that if this method returns true, `this.Parent.Green` and `other.Parent.Green` will be different nodes. They could actually be literally the same syntax node from the same syntax tree.

Reply:
true.  i'll try to reword to be more accurate.